### PR TITLE
fea(): Added FP32_SOFTMAX env var description

### PR DESCRIPTION
**Pull request by imangohari1**
updated the env var for the softmax fp32.

### CODE REVIEW

**Comment by imangohari1** [comment]
@MohitIntel

**Review by jedwards-habana — CHANGES_REQUESTED** [review]
Minor grammatical changes.

File: PyTorch/vLLM_Tutorials/Benchmarking_on_vLLM/Online_Static/README_qwen-2.5-vl.md [hunk 1]
```diff
@@ -42,10 +42,12 @@ Two broad scenarios are described with different methods recommended to avoid OO
...
+| `VLLM_GRAPH_MULTIMODAL_PROMPT_RATIO` | Ratio of text vs video memory in the prefill graphs memory pool | Affected by `VLLM_GRAPH_RESERVED_MEM` and `VLLM_GRAPH_PROMPT_RATIO`                                                 | Higher values skew the prefill graphs memory in favor of text part of the prompt suitable when fewer images and more text and vice versa                                                               | 0.3                                       |
+| `VLLM_FP32_SOFTMAX`                  | To enable FP32 softmax for the LLM model                        | Helps improving the accuray of the langugae model. Setting this to `=1` could have low/marginal performance impact. | `=0`                                                                                                                                                                                                   |
```
[the remark this answers is not in the document]
Comment (imangohari1) [reply PyTorch/vLLM_Tutorials/Benchmarking_on_vLLM/Online_Static/README_qwen-2.5-vl.md]: Done.

File: PyTorch/vLLM_Tutorials/Benchmarking_on_vLLM/Online_Static/README_qwen-2.5-vl.md [hunk 2]
```diff
@@ -42,10 +42,12 @@ Two broad scenarios are described with different methods recommended to avoid OO
...
+| `VLLM_FP32_SOFTMAX_VISION`           | To enable FP32 softmax for the vision model                     | Helps improving the accuray of the vision model. Setting this to `=1` could have noticable performance impact.      | `=0`                                                                                                                                                                                                   |
```
[the remark this answers is not in the document]
Comment (imangohari1) [reply PyTorch/vLLM_Tutorials/Benchmarking_on_vLLM/Online_Static/README_qwen-2.5-vl.md]: Done.

**Review by jedwards-habana — APPROVED** [review]
LGTM.
